pkg/steward/steward_test.go

```diff
@@ -29,6 +29,7 @@ func TestSteward(t *testing.T) {
 		data           = make([]byte, chunks*4096) //1k chunks
 		store          = mock.NewStorer()
 		traverser      = traversal.New(store)
+		loggingStore   = &loggingStore{Storer: store}
```

Comment:
Variable `loggingStore` shadows `loggingStore` type in the test. It does not create problems now, bit it would be nice not to shadow it.

Reply:
Done.